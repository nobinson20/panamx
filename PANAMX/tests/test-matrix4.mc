int main() {
    matrix mat;
    mat = <3, 3>;
    prints("original matrix:");

    matrix oth;
    oth = <4, 3>;

    mat[0][0] = 0;
    mat[0][1] = 1;
    mat[0][2] = 3;
    mat[1][0] = 0;
    mat[1][1] = 2;
    mat[1][2] = 3;
    mat[2][0] = 2;
    mat[2][1] = 4;
    mat[2][2] = 1;

    printm(mat);

    return 0;
}

