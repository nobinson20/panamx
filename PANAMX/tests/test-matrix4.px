int main() {
    matrix m;
    matrix n;
    m = [1, 2; 3, 4];
    n = 8.5 + m;
    printm(n);
    m = n - 1.5;
    printm(m);
    n = m * 2;
    printm(n);
    m = n / 4;
    printm(m);
    return 0;
}
