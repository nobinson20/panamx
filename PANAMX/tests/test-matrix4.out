[
	9.500   10.500   
	11.500   12.500   
]
[
	8.000   9.000   
	10.000   11.000   
]
[
	16.000   18.000   
	20.000   22.000   
]
[
	4.000   4.500   
	5.000   5.500   
]
